src/api/status/public/index.html

```diff
@@ -177,10 +177,14 @@ <h6 class="font-weight-bolder mb-0">Dashboard</h6>
                 </div>
                 <div class="text-end pt-1">
                   <h6 class="mb-0 text-capitalize">Telescope</h6>
-                  <p class="text-sm mb-0 text-capitalize">Yearly Commits</p>
-                  <h4 class="mb-0" id="yearly-commits-telescope">&nbsp;</h4>
-                  <p class="text-sm mb-0 text-capitalize">Weekly Commits</p>
-                  <h4 class="mb-0" id="weekly-commits-telescope">&nbsp;</h4>
+                  <div id="yearly-commits-telescope-dom">
```

Comment:
I wouldn't include the suffix, `-dom`, since everything in the DOM is part of the DOM already.

Reply:
Oh okay, I will change the name.